### PR TITLE
[Refactor] BlockStorage/extensions&noauth -> EVS/

### PR DESCRIPTION
**Pull request by Aloento**
Move all API methods and related structures to a separate file.
Inline all `Extracts` and use the `extract` package instead.
Remove all Builders

Move to EVS

All related tests passed.
```
=== RUN   TestGet
--- PASS: TestGet (0.00s)
=== RUN   TestGetUsage
--- PASS: TestGetUsage (0.00s)
=== RUN   TestFullUpdate
--- PASS: TestFullUpdate (0.00s)
=== RUN   TestPartialUpdate
--- PASS: TestPartialUpdate (0.00s)
=== RUN   TestErrorInToBlockStorageQuotaUpdateMap
--- PASS: TestErrorInToBlockStorageQuotaUpdateMap (0.00s)
=== RUN   TestDelete
--- PASS: TestDelete (0.00s)
=== RUN   TestListStoragePoolsDetail
--- PASS: TestListStoragePoolsDetail (0.00s)
=== RUN   TestListServices
--- PASS: TestListServices (0.00s)
=== RUN   TestAttach
--- PASS: TestAttach (0.00s)
=== RUN   TestBeginDetaching
--- PASS: TestBeginDetaching (0.00s)
=== RUN   TestDetach
--- PASS: TestDetach (0.00s)
=== RUN   TestUploadImage
--- PASS: TestUploadImage (0.00s)
=== RUN   TestReserve
--- PASS: TestReserve (0.00s)
=== RUN   TestUnreserve
--- PASS: TestUnreserve (0.00s)
=== RUN   TestInitializeConnection
--- PASS: TestInitializeConnection (0.00s)
=== RUN   TestTerminateConnection
--- PASS: TestTerminateConnection (0.00s)
=== RUN   TestExtendSize
--- PASS: TestExtendSize (0.00s)
=== RUN   TestForceDelete
--- PASS: TestForceDelete (0.00s)
=== RUN   TestNoAuth
--- PASS: TestNoAuth (0.00s)
```

### CODE REVIEW

File: openstack/blockstorage/v1/snapshots/util.go [hunk 1]
```diff
@@ -8,7 +8,7 @@ import (
 // status. It will do this for the amount of seconds defined.
 func WaitForStatus(c *golangsdk.ServiceClient, id, status string, secs int) error {
```
Comment (Aloento) [inline openstack/blockstorage/v1/snapshots/util.go]: Do we need util.go?

**Comment by Aloento** [comment]
There is a lot of duplicate code in V1 / 2 / 3, and much of it is not even changed. 
Can we consider merging them?

@outcatcher @anton-sidelnikov

**Comment by Aloento** [comment]
recheck

**Comment by anton-sidelnikov** [comment]
> There is a lot of duplicate code in V1 / 2 / 3, and much of it is not even changed. Can we consider merging them?
> 
> @outcatcher @anton-sidelnikov

We lost sdk structure if you merge it

**Comment by anton-sidelnikov** [comment]
Also it is too huge amount of changes, we cannot merge it at one time. Need to split this pr, and merge alongside with terraform

**Comment by Aloento** [comment]
We don't have to split this PR, we can open a branch on the TF side and change it.
It is also a lot of work to split this PR. @anton-sidelnikov

**Review by artem-lifshits — DISMISSED** [review]

File: openstack/evs/extensions/quotasets/testing/fixtures.go [hunk 2]
```diff
@@ -25,7 +26,7 @@ var getExpectedJSONBody = `
 	}
 }`
 
-var getExpectedQuotaSet = quotasets.QuotaSet{
+var getExpectedQuotaSet = quotasets2.QuotaSet{
```
Comment (anton-sidelnikov) [inline openstack/evs/extensions/quotasets/testing/fixtures.go]: Why here and below quotasets2?

File: openstack/evs/extensions/services/testing/requests_test.go [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+import (
+	"testing"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/openstack/evs/extensions/services"
+
+	"github.com/opentelekomcloud/gophertelekomcloud/testhelper"
+	"github.com/opentelekomcloud/gophertelekomcloud/testhelper/client"
+)
+
+func TestListServices(t *testing.T) {
```
Comment (anton-sidelnikov) [inline openstack/evs/extensions/services/testing/requests_test.go:12]: What exact checking this test?

**Review by anton-sidelnikov — CHANGES_REQUESTED** [review]

File: openstack/evs/extensions/services/testing/requests_test.go (hunk 3 above)
Comment (Aloento) [reply openstack/evs/extensions/services/testing/requests_test.go:12]: It just tests whether the method works or not with mock data.

**Review by anton-sidelnikov — APPROVED** [review]

**Review by artem-lifshits — APPROVED** [review]

**Pull request merged**